docs/getting-started.md

```diff
@@ -129,9 +129,3 @@ Now when we run `draft connect` and open the local URL using `curl` or our brows
 $ curl localhost:8080
 Hello, Draft!
 ```
-
-[Installation Guide]: install.md
```

Comment:
Sorry! Those other links are still important. I was just asking to remove this one. :)

Can you reinstate the others?

Reply:
@bacongobbler  OK.  I was wrong before.